js/reference.js

```diff
@@ -1669,13 +1669,33 @@
                 var column, keyColumns, referenceColumn;
 
                 // add column name into list of column projections if not in column projections set and data has changed
-                var addProjection = function(colName) {
+                var addProjection = function(colName, colType) {
                     // don't add a column name in if it's already there
                     // this can be the case for multi-edit
                     // and if the data is unchanged, no need to add the column name to the projections list
                     // NOTE: This doesn't properly verify if date/timestamp/timestamptz values were changed
-                    if ( (columnProjections.indexOf(colName) === -1) && (oldData[colName] != newData[colName]) ) {
-                        columnProjections.push(colName);
+                    if (columnProjections.indexOf(colName) === -1) {
```

Comment:
This is not that important but I feel like for these cases it's better if you use the reverse and just return here. That way you don't need to indent the code.

So

```js
if (columnProjections.indexOf(colName) !== -1) {
  return;
}

var typename = colType.rootName;
...

```


You should also update the comment above this line as you're now properly verifying date/timestamp/timetsamptz values.